README.md

```diff
@@ -1,2 +1,17 @@
 # smile-warrior
 This repo contains code used for smile detector for project of Gradient Science Club.
+
+## To create dataset:
+Run:
+```
+   python adjust_dataset.py 
+```
+
+Result will be a csv file with dataset for Smile-Warrior (by default in: "smile_warrior_dataset.csv").
+
+## To load dataset:
+Use:
+```
+   python data_server.py
+```
+Which returns respectively: x_train, y_train, x_test, y_test, x_validate and y_validate datasets. 
```

Comment:
What do you mean that script returns datasets?

Reply:
Do you mean that I should change it to something like "Which provides us with function that returns respectively: x_train, y_train, x_test, y_test, x_validate and y_validate datasets."?

Reply:
User does not need to know the function within script. He just runs:
```
python data_server.py
```
Why mention function at all?

Reply:
Ok, I removed info about function and rewrote it more generally